From: Colin Walters <walters@verbum.org>
Date: Wed, 8 Jun 2022 16:27:30 -0400
Subject: [PATCH] lib: Fix symbol versioning inheritance

I messed this up; the last release should inherit from the previous
release (N-1) and not the previous to that (N-2).

I think (hope) this isn't an ABI break...

Just noticed this when I was going to add a new symbol.

Applied-upstream: 2022.4, commit:145d91d1c96755bc61a468b5da1061547909121e
---
 src/libostree/libostree-released.sym | 2 +-
 tests/test-symbols.sh                | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/libostree/libostree-released.sym b/src/libostree/libostree-released.sym
index 8b8a722d8..5afdfd3e2 100644
--- a/src/libostree/libostree-released.sym
+++ b/src/libostree/libostree-released.sym
@@ -681,7 +681,7 @@ LIBOSTREE_2022.4 {
 global:
   ostree_fs_get_all_xattrs;
   ostree_fs_get_all_xattrs_at;
-} LIBOSTREE_2021.5;
+} LIBOSTREE_2022.2;
 
 /* NOTE: Only add more content here in release commits!  See the
  * comments at the top of this file.
diff --git a/tests/test-symbols.sh b/tests/test-symbols.sh
index a888ef358..a14849d56 100755
--- a/tests/test-symbols.sh
+++ b/tests/test-symbols.sh
@@ -54,7 +54,7 @@ echo 'ok documented symbols'
 
 # ONLY update this checksum in release commits!
 cat > released-sha256.txt <<EOF
-fb66f84d9af5270ed34fdf9f70f97e59394336a9559f493917704bdffa19027d  ${released_syms}
+34ff6c97e20242a449b210f7452a12d0bb6ac9e135638f71c2d27b2f2b583ef3  ${released_syms}
 EOF
 sha256sum -c released-sha256.txt
 
-- 
2.30.2

